kyaml/openapi/openapi.go

```diff
@@ -50,6 +52,13 @@ type openapiData struct {
 	schemaInit bool
 }
 
+type format string
+
+const (
+	JsonOrYaml format = "jsonOrYaml"
+	Proto      format = "proto"
```

Comment:
Why are these constants public but their type private?

Reply:
I didn't see a purpose in making the type public, but I can see a potential use case in the future of passing in the format from another package once we start supporting user-provided proto schemas, hence why I made the const public.

I'm happy to change them to both being public or both being private if that makes more sense, I don't have a strong preference either way.